net-mgmt/net-snmp/src/opnsense/service/conf/actions.d/actions_netsnmp.conf

```diff
@@ -1,23 +1,23 @@
 [start]
-command:/usr/local/opnsense/scripts/OPNsense/Netsnmp/setup.sh;/usr/local/etc/rc.d/snmpd start
+command:/usr/local/opnsense/scripts/OPNsense/Netsnmp/setup.sh;/usr/local/etc/rc.d/snmpd start;/usr/local/etc/rc.d/snmptrapd start
```

Comment:
@mimugmail is this even possible?

Reply:
we do need a separate service. stacking two daemons in one command sequence is asking for trouble

Reply:
How would I go about this? Can there be more than one action for a plugin?
Are there any examples? Thank you!